.travis.yml

```diff
@@ -1,20 +1,13 @@
-language:
-  - c
+language: node_js
+
 os:
   - linux
   - osx
-sudo: false
-install:
-  - rm -rf ~/.nvm
-  - git clone https://github.com/creationix/nvm.git ~/.nvm
-  - source ~/.nvm/nvm.sh
-  - nvm install $NODE_VERSION
-  - node --version
-  - npm install
-script: npm test
+
 after_success: npm run coverage
-env:
-  - NODE_VERSION="0.10"
-  - NODE_VERSION="0.12"
-  - NODE_VERSION="node"
-  - NODE_VERSION="4"
+
+node_js:
+  - "0.10"
+  - "0.12"
+  - "4"
+  - "5"
```

Comment:
Can we keep `"node"` here (instead of `"5"`)? That will guarantee we always test with the latest version available in nvm.